src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
+        colorSupplier.getRandomColor();
+        Lottery [] lotteries = new Lottery[3];
+        for (int i = 0; i < lotteries.length; i++) {
+            lotteries[i] = new Lottery();
+            System.out.println(lotteries[i].getRandomBall());
+        }
```

Comment:
You need only one Lottery and it should create three balls:
```suggestion
    Lottery lottery = new Lottery();
    for (int i = 0; i < 3; i++) {
        System.out.println(lottery.getRandomBall());
    }
```